ioctl/cmd/action/action.go

```diff
@@ -448,3 +456,16 @@ func isBalanceEnough(address string, act action.SealedEnvelope) error {
 	}
 	return nil
 }
+
+// getChainID returns the user input chainID
+func getChainID() uint32 {
+	var chainID uint32
+	endpoint := config.ReadConfig.Endpoint
```

Comment:
with the api.ChainID, we can query the chainID from api

Reply:
How about put it in `GetChainMetaResponse`, no need to add a new API.

Reply:
update proto file ,see PR
 https://github.com/iotexproject/iotex-proto/pull/116

Reply:
why add a new API?
we can use existing `GetChainMeta`

Reply:
1. i think can revert the `GetChainID` in iotex-proto, go back to use v0.5.2
2. also need change in ioctl to query chainID, and use it in `sendAction`